tests/test_hookman_generator.py

```diff
@@ -236,6 +237,22 @@ def test_generate_plugin_package(acme_hook_specs_file, tmpdir, mock_plugin_id_fr
     assert "assets/README.md" in list_of_files
     assert f"artifacts/{shared_lib_name}" in list_of_files
 
+    yaml_filename = tmpdir / plugin_id / 'assets' / 'plugin.yaml'
+    with open(yaml_filename, 'r') as f:
+        contents = f.read()
+
+    from textwrap import dedent
+    assert contents == dedent('''\
```

Comment:
Can we test overriding the defaults?

Reply:
I didn't get what you mean, this test is already using non-default values (from within hookman, `extras` is always empty, just `alfasim-sdk` will set the default required version).

Reply:
I meant that we should test that we are handling defaults correctly: we should have a plugin which overrides `extras`, another one which doesn't so the extras are passed unchanged.

Reply:
I've changed the test to have multiple _extras keys_, one overriding a default, one being added as new key and another default not being overriden. Is that good enough?

Reply:
Yep, thanks!